tools/cockpit.spec

```diff
@@ -107,10 +111,13 @@ Patch102:       0002-selinux-temporary-remove-setroubleshoot-section.patch
 %endif
 
 # Ship custom SELinux policy (but not for cockpit-appstream)
-%if 0%{?rhel} >= 9 || 0%{?fedora} || 0%{?suse_version}
+# SLES / Leap 15 does not have selinux-policy. TW and SLE Micro does.
+%define with_selinux 0
+%if 0%{?rhel} >= 9 || 0%{?fedora} || 0%{?suse_version} > 1600 || 0%{?is_smo}
 %if "%{name}" == "cockpit"
 %define selinuxtype targeted
-%define selinux_configure_arg --enable-selinux-policy=%{selinuxtype}
+%define with_selinux 1
+%define selinux_policy_version %(rpm --quiet -q selinux-policy && rpm -q --queryformat "%{V}-%{R}" selinux-policy || echo 1)
```

Comment:
the `%_selinux_policy_version` comes from `selinux-policy`, so there is no need to redefine it here or to remove the underscore, especially since it only applies to case with selinux

best to remove this line

Reply:
removed